docs/docsite/rst/dev_guide/developing_modules_general_windows.rst

```diff
@@ -190,6 +190,62 @@ A very basic powershell module `win_environment <https://github.com/ansible/ansi
 
 A slightly more advanced module is `win_uri <https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/windows/win_uri.ps1>`_ which additionally shows how to use different parameter types (bool, str, int, list, dict, path) and a selection of choices for parameters, how to fail a module and how to handle exceptions.
 
+As part of the new ``AnsibleModule`` wrapper, the input parameters are defined and validated based on an argument
+spec. The following options can be set at the root level of the argument spec:
+
+- ``mutually_exclusive``: A list of lists, where the inner list contains module options that cannot be set together
```

Comment:
These are comprehensive and helpful. Adding examples would make them even better. Could we add examples, or link to an existing Windows module that uses all or most of these options?

Reply:
Currently not all of these are used, I've just changed a few modules to use the new framework. The win_environment and win_uri module are linked in this doc as examples but as time goes on we will have more to reference to.